modules/installation-configuration-parameters.adoc

```diff
@@ -537,6 +578,14 @@ ifdef::ibm-power,ibm-power-vs[]
 |Determines the instruction set architecture of the machines in the pool. Currently, heterogeneous clusters are not supported, so all pools must specify the same architecture. Valid values are `ppc64le` (the default).
 |String
 endif::ibm-power,ibm-power-vs[]
+
+ifdef::agent[]
+|controlPlane:
+  architecture:
+|Determines the instruction set architecture of the machines in the pool. Currently, clusters with varied architectures are not supported. All pools must specify the same architecture. Valid values are `amd64` and `arm64`.
```

Comment:
see previous "values" comment

Reply:
Based on the way this attribute is written for other install pages (see lines 551-580) I guess not. I'd prefer to follow the existing format unless you'd prefer me to put these values in the other column.